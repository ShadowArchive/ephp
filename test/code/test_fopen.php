<?php

$f = fopen(__FILE__, "r");
$h = fread($f, 5);
fclose($f);

?>
[<?=$h?>]

<?php

fopen("noexiste", "r");
fclose($f);

