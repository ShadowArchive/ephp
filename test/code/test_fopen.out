[<?php]


Warning: fopen(noexiste): failed to open stream: No such file or directory in {{CWD}}/test/code/test_fopen.php on line 12

Warning: fclose(): 1 is not a valid stream resource in {{CWD}}/test/code/test_fopen.php on line 13
